### PR TITLE
fix: dev server not working

### PR DESCRIPTION
**Pull request by maltoze**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "dev-tauri-renderer": "vite -c vite.config.tauri.ts",
     "build-tauri": "npm run build-tauri-renderer && tauri build",
     "dev-tauri": "tauri dev",
-    "dev-chromium": "NODE_ENV=development vite build -c vite.config.chromium.ts --watch",
+    "dev-chromium": "vite -c vite.config.chromium.ts",
     "dev-firefox": "NODE_ENV=development vite build -c vite.config.firefox.ts --watch",
```
Comment (yetone) [inline package.json:12]: this command can be changed to the real dev command?

**Comment by yetone** [comment]
image

**Comment by maltoze** [comment]
> image

fixed

File: package.json (hunk 1 above)
Comment (maltoze) [reply package.json:12]: what's the "real dev command"?

Comment (yetone) [reply package.json:12]: like dev-chromium

Comment (maltoze) [reply package.json:12]: firefox not support that 😂
https://github.com/samrum/vite-plugin-web-extension/issues/87#issuecomment-1501236239

**Comment by yetone** [comment]
great work!

**Comment by yetone** [comment]
Encountered an error 

image

**Comment by maltoze** [comment]
@yetone just removed this plugin https://github.com/openai-translator/openai-translator/pull/915

**Pull request merged**